### PR TITLE
Remove additional CloudStack Multiple WorkerNodeGroup checks

### PR DESCRIPTION
**Pull request by maxdrib**
*Issue #, if available:*

*Description of changes:*
Inadvertently found more restrictions on multiple worker node groups which were not detected previously. Since this feature is supported now, removing the extra checks.

*Testing (if applicable):*
Unit tests pass

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by vignesh-goutham — APPROVED** [review]
I'll wait for @vivek-koppuru to take a look and approve

File: pkg/api/v1alpha1/cluster_webhook_test.go [hunk 1]
```diff
@@ -1242,6 +1242,7 @@ func TestClusterCreateManagementCluster(t *testing.T) {
 
 func TestClusterCreateCloudStackMultipleWorkerNodeGroupsValidation(t *testing.T) {
 	os.Setenv(features.CloudStackProviderEnvVar, "true")
+	os.Setenv("FULL_LIFECYCLE_API", "true")
```
Comment (vivek-koppuru) [inline pkg/api/v1alpha1/cluster_webhook_test.go:1245]: I would suggest setting the env var this way:

https://github.com/aws/eks-anywhere/blob/main/pkg/clustermanager/cluster_manager_test.go#L932

Comment (maxdrib) [reply pkg/api/v1alpha1/cluster_webhook_test.go]: Fixed, thank you. I've been struggling with this for a while now

**Review by vivek-koppuru — APPROVED** [review]
/approve

**Comment by vivek-koppuru** [comment]
Overriding project unit test check because no new code

**Pull request merged**